### ISSUE TITLE
Autoplay is not allowed...

### ISSUE THREAD

**Issue by elmatou** [jamesbachini/StayAwake.js#4]
Hi there,
currently running Firefox 78.0.1 (but similar warning exists for current Opera and Chrome)
The library do not work and following errors and message are issued :

```
Autoplay is only allowed when approved by the user, the site is activated by the user, or media is muted.
```
and
```
Uncaught (in promise) DOMException: The play method is not allowed by the user agent or the platform in the current context, possibly because the user denied permission.
```
or in Opera 
```
Uncaught (in promise) DOMException: play() failed because the user didn't interact with the document first.
```

The current Google policy is detailed here: https://developers.google.com/web/updates/2017/09/autoplay-policy-changes

I can confirm that a muted video is allowed to run in background.
```
this.stayAwakeVideo.muted = true;
```
did the trick, the mistake is to believe that the HTML attribute `muted` works the same way.

**Comment by jamesbachini** [comment]
hi elmatou,

Thanks for this. I've added in the muted=true line below the setAttribute('muted') line. It's a while ago since I wrote the code and seem to remember a previous issue which led to using setAttribute. I've left the original in along with the new line. Can't foresee any downside of having them both in there and seems to work in my tests currently.

Thanks again for your help.

James

Closed Ref: https://github.com/jamesbachini/StayAwake.js/commit/cb3a16cd07aa2776f904ac74fdeca1a52f9229a9
